install/installation-kubernetes.md

```diff
@@ -0,0 +1,150 @@
+[TimescaleDB on Kubernetes][timescaledb-k8s].
+You can install TimescaleDB on Kubernetes deployed on:
+* [AWS Elastic Kubernetes Service][aws-eks]
+* [MicroK8s][microk8s-install]
+* [minikube][minikube-install]
+
+Before you begin installing TimescaleDB on a Kubernetes deployment, make sure you have:
+* Installed [kubectl][kubectl-install]
+* Installed [Helm][helm-install]
+* Installed [Kubernetes Cluster][kubernetes-install]
+* (**Optional**) Created your own `.yaml` file if you want to use parameters other than those specified in the default `values.yaml`. You can name this file `myvalues.yaml` file. For details about the parameters you can set, see  the [Administrator Guide][admin-guide].
+
+
+## Install TimescaleDB using a Helm chart
+You can install TimescaleDB on Kubernetes using a Helm chart with the default `values.yaml` file. 
+When you use the default `values.yaml`, the user credentials are randomly generated during installation.
+When you then use the `helm upgrade` command, it does not rotate the credentials, because changing the database credentials would break the database. Instead, it continues to use the credentials generated during `helm install`.
+
+This section provides instructions to deploy TimescaleDB using the `timescaledb-single` Helm chart.
+
+<procedure>
+
+### Installing TimescaleDB using a Helm chart
+1.  Add the TimescaleDB Helm chart repository:
+    ```bash
+    helm repo add timescale 'https://charts.timescale.com'
+    ```
+1.  Verify that the repository is up to date:
+    ```bash
+    helm repo update
+    ```
+1.  Install the TimescaleDB Helm chart, by replacing `<my_name>` with a name of your choice:
+    ```bash
+    helm install <my_name> timescale/timescaledb-single
+    ```
+    If you created a `<myvalues.yaml>` file, use this command instead:
+    ```bash
+    helm install <my_name> -f <myvalues.yaml> charts/timescaledb-single
+    ```
+
+</procedure> 
+
+## Connect to TimescaleDB
+You can connect to TimescaleDB from an external IP address, or from within the cluster.
+
+<procedure>
+
+### Connecting to TimescaleDB using an External IP
+
+<highlight type="note">
+If you configured the user credentials in the `my_values.yaml` file, you don't need to decode the passwords.
+</highlight>
+
+1. Get the name of the host to connect to:
+    ```bash
+    kubectl get service/my-release
+    ```
+1. Decode the `admin` user password `PGPASSWORD_ADMIN` that was generated during the Helm installation, by replacing `<my_name>` with the name that you provided during the installation:
+    ```bash
+    PGPASSWORD_ADMIN=$(kubectl get secret --namespace default <my_name>-credentials -o jsonpath="{.data.PATRONI_admin_PASSWORD}" | base64 --decode)
+    ``` 
+1. (**Optional**) Decode the super user password `PGPOSTGRESPASSWORD` that was generated during the Helm installation, by replacing `<my_name>` with the name that you provided during the installation:
```

Comment:
I think in other places, I've used two asterisks for an optional step, but no parentheses. Could be worth double checking, though.

Reply:
checked that it was **OPTIONAL**  and updated accordingly.